tools/BugReportTool/BugReportTool/BugReportTool.vcxproj

```diff
@@ -30,14 +30,15 @@
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
-      <AdditionalDependencies>Wevtapi.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>Version.lib;Wevtapi.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
It's not your change. Rebase on the current master. You can even close this pr and open a new one if it is easier this way.

Reply:
@mykhailopylyp I have done a rebase - I think - I'm still pretty new to git, so please let me know if this did not do what was needed.

Reply:
Merge `microsoft:master` branch into `chrisharris333:powertoyrun-confirm`. And then push it.

Reply:
```
src\modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.System\Main.cs(213,53): Error CS1061: 'PluginMetadata' does not contain a definition for 'ConfirmSystemCommands' and no accessible extension method 'ConfirmSystemCommands' accepting a first argument of type 'PluginMetadata' could be found (are you missing a using directive or an assembly reference?)
```

Reply:
λ git merge microsoft:master chrisharris333:powertoyrun-confirm
merge: microsoft:master - not something we can merge

Reply:
I was saying that you should merge upstream master into your branch. 
I hope the following article would be helpful 
https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/merging-an-upstream-repository-into-your-fork

Reply:
Article was helpful, thank you.  I think I got this done right this time.